source/help/apps/desktop-guide.rst

```diff
@@ -1,104 +1,95 @@
-===================================
 Using Mattermost Desktop App
-===================================
+============================
 
-Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
+Mattermost desktop applications are available for Windows, Mac and Linux operating systems.
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during setup and for troubleshooting tips.
 
 To view the latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
 
 Below are a few tips to get you started and to configure your experience on the desktop app:
 
- - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#id1>`_
- - `App Options <https://docs.mattermost.com/help/apps/desktop-guide.html#id2>`_
- - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#id3>`_
+.. contents::
+    :backlinks: top
+    :local:
 
-Team Management
----------------------------------------------------------------------
+Server Management
+-----------------
 
 You can connect to multiple Mattermost servers from a single interface on the desktop app.
 
-The Team Management section allows you to add, edit and remove servers and teams.
+The server Management section allows you to add, edit, and remove servers.
 
-Adding servers and teams
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adding servers
+~~~~~~~~~~~~~~
 
-To add a new server on the Mattermost instance, 
+To add a new server on the Mattermost instance:
```

Comment:
What Mattermost instance? Propose: "To access a new server from your desktop app environment"